x-pack/plugins/cases/server/services/user_actions/index.ts

```diff
@@ -691,126 +626,3 @@ export class CaseUserActionService {
     };
   }
 }
-
-export function transformFindResponseToExternalModel(
```

Comment:
All these were moved to `transform.ts` since they're needed here and in `find.ts`.